### ISSUE TITLE
Per colour channel saturation setting

### ISSUE THREAD

**Issue by mightysashiman** [averne/Fizeau#67]
On the OLED switch, some games (Sonic Superstar comes to mind) have colours that in “vivid” colour mode looks imho oversaturated. Using Fizeau to set overall saturation to 0.90 helps somewhat: reds look better, blue look still look oversaturated, while greens look fine. Pretty sure it is entirely up to personal preference. Would it be possible to implement more precise per-channel (R, G, B) saturation settings please?

**Comment by sauliiin** [comment]
Also, sorry to uses this channel to made a request, but since now screen overclock is a thing, is it possible to add the possibility to change profiles on overlay menu?

**Comment by mightysashiman** [comment]
bump, I'm also seeing the need for per-channel correction in the recent Zelda Echoes of Wisdom, which to my taste could benefit from a slight undersaturation of greens. Would you mind considering this additional feature please?

**Comment by averne** [comment]
I implemented this in a more general way: color correction can now be enabled selectively to color channels. So for instance, if you wanted to desaturate the greens, you would pick a saturation value under one, and enable only the green channel. 

Here's a build, I'm gonna test it a bit further to see if I broke anything before doing a release.
[Fizeau-2.7.0-d9802ba.zip](https://github.com/user-attachments/files/17445151/Fizeau-2.7.0-d9802ba.zip)
